code/modules/unit_tests/dcs_check_list_arguments.dm

```diff
@@ -0,0 +1,58 @@
+/**
+ * list arguments for bespoke elements are treated just like any other datum: as a text ref in the ID.
+ * Using un-cached lists in AddElement() and RemoveElement() calls will just create new elements over
+ * and over. That's what this unit test is for. It's not a catch-all, but it does a decent job at it.
+ */
+/datum/unit_test/dcs_check_list_arguments
```

Comment:
Does this need its own test, or can we make an assert in AddElement itself instead?

Reply:
Change "can" to "is it easier"

Reply:
> Does this need its own test, or can we make an assert in AddElement itself instead?

I'd rather have it go through and compare all lists in one big swoop than throw several stack traces, it's more succinct. Anyhow, I've offloaded the sorting part of the unit test to the dcs.dm file. This way the unit test doesn't have to populate yet another list.